stdlib/pathlib.pyi

```diff
@@ -53,15 +59,21 @@ class PurePath(PathLike[str]):
     def as_uri(self) -> str: ...
     def is_absolute(self) -> bool: ...
     def is_reserved(self) -> bool: ...
-    if sys.version_info >= (3, 9):
+    if sys.version_info >= (3, 12):
+        def is_relative_to(self, other: StrPath, /, *_deprecated: StrPath) -> bool: ...
```

Comment:
We can't use PEP 570 syntax yet, even if it didn't break our tests, since we still support Python 3.7.

```suggestion
        def is_relative_to(self, other: StrPath, *__deprecated: StrPath) -> bool: ...
```

(Might need a stubtest allowlist entry.)

Reply:
The argument before the slash needs to have the `__` prefix, not the one after

Reply:
I don't know what you mean. 😇